.codecov.yml

```diff
@@ -0,0 +1,20 @@
+# Automatically generated by nengo-bones, do not edit this file directly
+# Version: 0.1.0.dev0
```

Comment:
Can this be included as yml data to make parsing it less hacky?

Reply:
We could, but then we'd need to have different version checking methods for different files (not all of them will be `.yml` files).  We'd also be relying on the code that actual parses these files normally (codecov and TravisCI) to ignore these extra yml config values we've added, which I'm not sure whether they would do or not.

Reply:
Follow-up question: will all of the files that we plan on templating use the `#` syntax to denote comments? If not, then we may need a custom strategy that depends on the file extension anyway.